docs/cmakelists.md

```diff
@@ -36,7 +36,8 @@ related to making Python extension modules.
 
 If you are making a Limited API / Stable ABI package, you'll need the
 `Development.SABIModule` component instead (CMake 3.26+). You can use the
-`SKBUILD_LIMITED_API` variable to check to see if it was requested.
+`Python_Development.SABIModule_FOUND` variable to check to see if it was
```

Comment:
```suggestion
`SKBUILD_LIMITED_ABI` variable to check to see if it was
```

Reviewing changes introduced in https://github.com/scikit-build/scikit-build-core/pull/333/files, I just realized that it should have been `SKBUILD_LIMITED_ABI ` instead of `SKBUILD_LIMITED_API `